plugins/modules/nios_network.py

```diff
@@ -287,7 +312,8 @@ def main():
         template=dict(type='str'),
         extattrs=dict(type='dict'),
         comment=dict(),
-        container=dict(type='bool', ib_req=True)
+        container=dict(type='bool', ib_req=True),
+        members=dict(type='list', elements='dict')
     )
```

Comment:
Hi @matthewdennett , QA have some concerns or bugs over this PR. I am listing them over here.  Please take a look , if you can handle this.

1. Unable to remove member assignment from a network
    1.1 they have tried to re-execute playbook without members field, but members assigned are not being deleted
2. Unable to create a network container with members assignment feature

3. Unable to update master candidate field  for NIOS member
   3.1 After the member creation, with master candidate field set false. For updating the member as GMC is getting failed.
   
   These are some concerns for feature support, Please see if you could handle this minor additions and fixes

Reply:
Hi @hemanthKa677, 

Thanks for that update. I will have a look at the reported issues you have described. Is it possible for the QA team to provide a copy of the tests they ran? That will be helpful to test against while coding a fix.

Reply:
Hi @hemanthKa677, 

I have resolved the above mentioned bugs. Can you please send this off to the QA team for review?

Reply:
Hi @matthewdennett , I will test these changes and send to the QA team.